### PR TITLE
Added some more flavor to the README

### PR DESCRIPTION
**Pull request by TimonPost**
Added some more flavor to the amethyst network README

### CODE REVIEW

File: amethyst_network/README.md [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
 
-Right now this crate is very simple and it just supports sending messages through specs to other clients. It uses a combination of `EventChannel`s
-to send events between the network thread. In its current state it is not ready to be used in a game but there is work going into [Laminar](https://github.com/amethyst/laminar) that will soon be the underlying socket implemenation for this crate.
+The networking crate for the `amethyst` game engine. This crate provides the API and functionality which application developers will normally use to develop multiplayer games. The main engine can be found at https://amethyst.rs.
 
-For more information or help, please come find us on the discord server's #net channel.
+This project is at an early stage. We are currently designing and working on some robust, fast, distributed, networking system on top of specs. To make this work we are creating a small test game so that we can experiment with different solutions. Once in a while, we will move some stable functionality over from that game to amethyst network.
+
+Currently, amethyst network supports:
+- Reliable (ordered, sequenced) UPD
```
Comment (jstnlef) [inline amethyst_network/README.md]: Reliable (ordered, sequenced) **UDP**

File: amethyst_network/README.md [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+
+Unless you explicitly state otherwise, any Contribution intentionally submitted for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any
+additional terms or conditions.
+
+For more information or help, please come find us on the amethyst discord server's `#net` channel. We are working on architecture, design, and roadmaps and can definitely use some helping hands, don't hessitate :). 
+
+## License
+
+Licensed under either of
+ * Apache License, Version 2.0 ([LICENSE-APACHE](docs/LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
```
Comment (azriel91) [inline amethyst_network/README.md]: nitpicky -- the URLs can be `https` (checked)

File: amethyst_network/README.md [hunk 3]
```diff
@@ -1,10 +1,41 @@
...
+[s5]: https://img.shields.io/discord/425678876929163284.svg?logo=discord
+[l5]: https://discord.gg/GnP5Whs
 
-Right now this crate is very simple and it just supports sending messages through specs to other clients. It uses a combination of `EventChannel`s
-to send events between the network thread. In its current state it is not ready to be used in a game but there is work going into [Laminar](https://github.com/amethyst/laminar) that will soon be the underlying socket implemenation for this crate.
+The networking crate for the `amethyst` game engine. This crate provides the API and functionality which application developers will normally use to develop multiplayer games. The main engine can be found at https://amethyst.rs.
 
-For more information or help, please come find us on the discord server's #net channel.
+This project is at an early stage. We are currently designing and working on some robust, fast, distributed, networking system on top of specs. To make this work we are creating a small test game so that we can experiment with different solutions. Once in a while, we will move some stable functionality over from that game to amethyst network.
```
Comment (jstnlef) [inline amethyst_network/README.md]: "This project is still at an early stage. We are currently designing and implementing a fast/robust networking system on top of specs. To exercise our implementation, we are creating a small test game which we will make public when we feel it's in a good place. Eventually, as we gain more confidence in our solution, we will move stable functionality over from that game to amethyst network."

Comment (jstnlef) [reply amethyst_network/README.md]: Or something to that effect.

Comment (TimonPost) [reply amethyst_network/README.md]: Good one!

**Pull request merged**